soh/soh/Enhancements/RogueLike/ActorBehavior/Enemies.cpp

```diff
@@ -36,7 +36,12 @@ static void InitEnemyBehavior() {
 
     COND_HOOK(OnActorUpdate, IS_ROGUELIKE, [](void* actor) {
         Actor* refActor = static_cast<Actor*>(actor);
-        if (refActor->category != ACTORCAT_ENEMY) {
+        if (refActor->category != ACTORCAT_ENEMY && refActor->id != ACTOR_OBJ_TSUBO) {
+            return;
+        }
+
+        if (refActor->id == ACTOR_OBJ_TSUBO && refActor->params == 256) {
+            Actor_SetColorFilter(refActor, 0x1000, 150, 0, 1000);
```

Comment:
Since pots return early anyway, if you swap these two, it saves you from having to check a whole category for one thing:
```suggestion
        if (refActor->id == ACTOR_OBJ_TSUBO && refActor->params == 256) {
            Actor_SetColorFilter(refActor, 0x1000, 150, 0, 1000);
            return
        }

        if (refActor->category != ACTORCAT_ENEMY) {
```